src/plugins/condition/components/TestData.vue

```diff
@@ -200,6 +201,10 @@ export default {
           this.telemetryMetadataOptions[id] = [];
         }
       });
+      this.telemetry.forEach(async (telemetryOption, index) => {
```

Comment:
async in a forEach is usually a problem. can we instead use a map and Promise.all them?

e.g.:
```js
async updateTelemetryPaths() {
  this.telemetryPaths = await Promise.all(
    this.telemetry.map(async (telemetryOption) => {
      return await this.getFullTelemetryPath(telemetryOption);
    })
  );
```